extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -9,14 +9,41 @@ define('ext.wikia.adEngine.template.bfaa', [
 ], function (adContext, adHelper, doc, log, win, mercuryListener) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.template.bfaa',
+	var bfabSlotName = 'BOTTOM_LEADERBOARD',
 		breakPointWidthNotSupported = 767, // SCSS property: $breakpoint-width-not-supported
 		desktopHandler,
+		disableBtf = win.ads.runtime.disableBtf,
```

Comment:
We've got the `ext.wikia.adEngine.provider.btfBlocker` which is coupled with `win.ads.runtime.disableBtf`. And I would keep it this way: to keep this dependency in one place. Let's think of how to make it.